packages/block-library/src/button/edit.js

```diff
@@ -54,6 +62,27 @@ class ButtonEdit extends Component {
 		this.nodeRef = node;
 	}
 
+	onSetLinkRel( value ) {
+		this.props.setAttributes( { rel: value } );
+	}
+
+	onSetNewTab( value ) {
```

Comment:
Nitpick: I think the name of this method could be a bit more verbose, maybe `onToggleOpenInNewTab`?

Reply:
Had picked that up from the `image` block 😛 changed in [this commit](https://github.com/WordPress/gutenberg/pull/10128/commits/ba67c69330fd7d8221341f31de1e2b6cd6ef6eed). Thank you @talldan ❤️